colossus/src/main/scala/colossus/service/ServiceClient.scala

```diff
@@ -92,7 +93,7 @@ object ClientConfig {
   }
 }
 
-class ServiceClientException(message: String) extends Exception(message)
+class ServiceClientException(message: String) extends Exception(message) with ColossusRuntimeException
```

Comment:
this is where we're squashing the stracktrace, we should pass it along here

Reply:
Can we get an example of the error that is happening so we can see if the stack trace is helpful, because for all these `ColossusRuntimeException` the stack trace shouldn't be helpful.